bugsnag-android-core/src/main/java/com/bugsnag/android/SessionTrackingPayload.java

```diff
@@ -17,11 +17,12 @@
 
     SessionTrackingPayload(Session session,
                            List<File> files,
-                           AppData appData,
-                           DeviceData deviceData) {
-        this.appDataSummary = appData.getAppDataSummary();
-        this.deviceDataSummary = deviceData.getDeviceDataSummary();
-        this.notifier = Notifier.getInstance();
+                           Map<String, Object> appDataSummary,
```

Comment:
While I understand these are fairly minor changes (i.e. just changing where the Object -> Map conversion is), could someone else (@kattrali ?) review any changes to the core SDK?

Are we certain there are no places that call this function that could have been missed?